stats/stats.go

```diff
@@ -165,6 +165,8 @@ type OutTrailer struct {
 	// Client is true if this OutTrailer is from client side.
 	Client bool
 	// WireLength is the wire length of trailer.
+	// Deprecated: This field is never set. The length is not known when this message is
+	//             emitted because the trailer fields are compressed with hpack after that.
```

Comment:
Please remove the whitespaces between `//` and `emitted`; this is not the convention we've been following in our repo.